app/assets/stylesheets/moj/_pq.scss

```diff
@@ -426,6 +426,11 @@ textarea.form-control {
     margin-bottom: 5px;
   }
 
+  .govuk-heading-m {
```

Comment:
I'm not sure this style should be changed. 
Maybe add styles for just span.question-uin?